dspace-server-webapp/src/main/java/org/dspace/app/rest/converter/ItemConverter.java

```diff
@@ -51,6 +63,18 @@ public ItemRest convert(Item obj, Projection projection) {
         item.setWithdrawn(obj.isWithdrawn());
         item.setLastModified(obj.getLastModified());
 
+        try {
+            Entity entity = entityService.findByItemId(contextService.getContext(), UUID.fromString(item.getId()));
+            EntityType entityType = entityService.getType(contextService.getContext(), entity);
+            if (entityType != null) {
+                item.setEntityType(entityType.getLabel());
+            }
```

Comment:
due to the performance noted for the sitemaps generator I assume that this operation has an impact, why not simply  return the value of the relationship.type metadata? we know that it represent our entity type label and if later on this rule will change the added test will warn us

Reply:
It was done to avoid repeating the relationship.type metadata field name (especially since it's planned to be renamed)
The ITs do indeed indicate it as a problem if this would change, so it can simply return that metadata